sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -0,0 +1,40 @@
+## API Report File for "@azure/container-registry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export class ContainerRegistryClient {
+    constructor(endpointUrl: string, credential: TokenCredential, options?: ContainerRegistryClientOptions);
+    getAttributes(name: string, options?: GetAttributesOptions): Promise<RepositoryAttributes>;
+}
+
+// @public
+export interface ContainerRegistryClientOptions extends PipelineOptions {
+}
+
+// @public
+export interface GetAttributesOptions extends OperationOptions {
+}
+
+// @public
+export interface RepositoryAttributes {
+    // Warning: (ae-forgotten-export) The symbol "ChangeableAttributes" needs to be exported by the entry point index.d.ts
```

Comment:
Looks like we're missing a symbol!

Reply:
Yeah, The library is not in any working state yet so I didn't want to spend too much time on moving parts.

Reply:
Fixed